Makefile

```diff
@@ -1,36 +1,21 @@
-#
-#	Build usable docs
-#
+HEADER_SOURCE := header.adoc
+PDF_RESULT := RISCV_ACPI_PLATFORM-spec.pdf
 
-ASCIIDOCTOR = asciidoctor
-DITAA = ditaa
-RISCV_ACPI_SPEC = riscv-acpi-platform-req
-PANDOC = pandoc
+all: build
 
-# Build the platform spec in several formats
-all: $(RISCV_ACPI_SPEC).md $(RISCV_ACPI_SPEC).pdf $(RISCV_ACPI_SPEC).html
+build:
 
-$(RISCV_ACPI_SPEC).md: $(RISCV_ACPI_SPEC).xml
```

Comment:
The RISC-V structure is only for building PDFs.  As such, building XML, HTML, and other targets need not be removed.  Support and/or removal is a community decision.

Reply:
Thanks. This was carried from another makefile template. PDF should be sufficient for now.